R/expect_lint.R

```diff
@@ -57,13 +57,14 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
   n_lints <- length(lints)
   lint_str <- if (n_lints) paste0(c("", lints), collapse = "\n") else ""
 
-  wrong_number_fmt  <- "got %d lints instead of %d%s"
+  wrong_number_fmt <- "got %d lints instead of %d%s"
   if (is.null(checks)) {
     msg <- sprintf(wrong_number_fmt, n_lints, length(checks), lint_str)
     return(testthat::expect(n_lints %==% 0L, msg))
   }
 
-  if (!is.list(checks) || !is.null(names(checks))) { # vector or named list
+  if (!is.list(checks) || !is.null(names(checks))) {
```

Comment:
is this a tidyverse rule?

Reply:
I don't think so. I will revert this change; definitely unwanted for our purposes.

Reply:
is there a styler bug to report?

Reply:
Yes, this actually messes with lintr directives in comments.

Here is a minimal and compelling reprex:

``` r
styler::style_text(
  "T_and_F_symbol_linter <- function() { # nolint: object_name.
  }"
)
#> T_and_F_symbol_linter <- function() {
#>   # nolint: object_name.
#> }
```

<sup>Created on 2022-09-19 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
Would you like to create an issue or should I?

Reply:
go ahead since you've got the reprex already, thanks

Reply:
Done: https://github.com/r-lib/styler/issues/1004